neofetch

```diff
@@ -3536,18 +3530,8 @@ get_full_path() {
 }
 
 get_default_config() {
-    if [[ -f "/etc/neofetch/config" ]]; then
-        default_config="/etc/neofetch/config"
-
-    elif [[ -f "/usr/local/etc/neofetch/config" ]]; then
-        default_config="/usr/local/etc/neofetch/config"
-
-    elif [[ -f "/data/data/com.termux/files/usr/etc/neofetch/config" ]]; then
-        default_config="/data/data/com.termux/files/usr/etc/neofetch/config"
-
-    elif [[ -f "/boot/home/config/non-packaged/etc/neofetch/config" ]]; then
-        default_config="/boot/home/config/non-packaged/etc/neofetch/config"
-
+    if [[ -f "${NEOFETCH_PREFIX}/neofetch/config" ]]; then
+        default_config="${NEOFETCH_PREFIX}/neofetch/config"
```

Comment:
This will check the default config file in `/usr` or whatever `$NEOFETCH_PREFIX` should be instead of `/etc`.